### PR TITLE
[12.0][MIG] stock_picking_restrict_cancel_with_orig_move

### PR DESCRIPTION
**Pull request by kittiu**
Migrate to v12

### CODE REVIEW

File: stock_picking_restrict_cancel_with_orig_move/__manifest__.py
Comment (rousseldenis) [inline stock_picking_restrict_cancel_with_orig_move/__manifest__.py]: @kittiu When migrating, we reset the trailing version. So, '12.0.1.0.0'

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@kittiu Could you look at travis? It seems you forgot to migrate uom references

File: stock_picking_restrict_cancel_with_orig_move/__manifest__.py
Comment (kittiu) [reply stock_picking_restrict_cancel_with_orig_move/__manifest__.py]: Yes, sorry, I didn't complete this one yet. It is in progress. I create PR just to say I will be doing it + to see travis log. 
Please help mark it "In Progress"

Comment (kittiu) [reply stock_picking_restrict_cancel_with_orig_move/__manifest__.py]: Currently, there are 4 addons I am working on. Sorry for the confusion.

Comment (rousseldenis) [reply stock_picking_restrict_cancel_with_orig_move/__manifest__.py]: No problem. Don't hesitate to mark 'WIP' in comment when creating it.

**Comment by kittiu** [comment]
Migrated

**Comment by kittiu** [comment]
Hi, after this one is merged, then we can finish **purchase_stock_picking_restrict_cancel** :)

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by kittiu** [comment]
@rousseldenis 
When we migrate to 12.0, should the version reset to 12.0.1.0.0 ? (now it is 12.0.1.0.2)

**Comment by rousseldenis** [comment]
> When we migrate to 12.0, should the version reset to 12.0.1.0.0 ? (now it is 12.0.1.0.2)

Yes.

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration

**Comment by kittiu** [comment]
> > When we migrate to 12.0, should the version reset to 12.0.1.0.0 ? (now it is 12.0.1.0.2)
> 
> Yes.
> 
> https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration

ok, changed.

**Review by LoisRForgeFlow — APPROVED** [review]
code review LGTM :+1:

**Pull request merged**
